nerfstudio/cameras/cameras.py

```diff
@@ -22,13 +22,16 @@
 from typing import Dict, List, Optional, Tuple, Union
 
 import cv2
+import nerfacc
```

Comment:
Im a little cautious about importing this here. Currently nerfacc is only used by instant-ngp which means that those cuda kernels were only getting build if people were using that model. Adding it here would force everyone to build those kernels (and some have had issues with that in the past). Not really sure what the best option is...

Reply:
Correct me if I am wrong but doesn't nerfacc builds the kernel only on the actual use of one of its method?
If not, we can import it dynamically in the function using importlib

https://www.geeksforgeeks.org/importlib-package-in-python/

Reply:
just for the sport of it, I implemented the interction with aabb box in pure pytorch. 
I added tests and check that it produces the same outputs as nerfacc for random set of rays.
It is order of magnitude slower and the nerfacc (For 500 rays ~ 0.4 milisec . nerfacc ~ 0.05 milisec)  , 
but it is still pretty usable in the gui.

I define env variable "INTERSECT_WITH_NERFACC" with default of 1 -
if INTERSECT_WITH_NERFACC is set to 1, we use nerfacc with dynamic import.
if INTERSECT_WITH_NERFACC is set to 0, we can use the pytorch alternative.

users that will have problem with INTERSECT_WITH_NERFACC can be instructed to set it to 0.